### PR TITLE
adds users

### PR DESCRIPTION
**Pull request by inferal73**

### CODE REVIEW

File: test/controllers/web/developers_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  test 'should get new' do
+    get :new
+    assert_response :success
+  end
+
+  test 'should post create' do
+    assert_difference 'Developer.count' do
+      post :create, params: { developer: attributes_for(:developer) }
+    end
+    assert_response :redirect
```
Comment (moklidia) [inline test/controllers/web/developers_controller_test.rb:13]: Let's also check that the user was created.

File: test/controllers/web/sessions_controller_test.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Web::SessionsControllerTest < ActionController::TestCase
       email: user.email,
       password: password,
     }
-    assert_difference 'Session.count' do
+    assert_difference 'session.count' do
```
Comment (moklidia) [inline test/controllers/web/sessions_controller_test.rb:16]: Why `session`, not `Session`?
Let's first add old_sessions_count and then assert that the Session.count == old_sessions_count + 1

File: test/controllers/web/developers_controller_test.rb (hunk 1 above)
Comment (moklidia) [inline test/controllers/web/developers_controller_test.rb:13]: Let's check that the user was actually created.

File: test/controllers/web/sessions_controller_test.rb (hunk 2 above)
Comment (inferal73) [reply test/controllers/web/sessions_controller_test.rb:16]: I use assert_difference to track differences in the number of entries. https://apidock.com/rails/ActiveSupport/Testing/Assertions/assert_difference.
With capital letter Ruby is looking for a model

File: test/controllers/web/developers_controller_test.rb (hunk 1 above)
Comment (inferal73) [reply test/controllers/web/developers_controller_test.rb:13]: I check the difference in quantity for the entity. If the user is not created, the number will not change.

Comment (moklidia) [reply test/controllers/web/developers_controller_test.rb:13]: Yep, sorry, didn't notice that assert.

File: app/controllers/web/developers_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  def create
+    @developer = Developer.new(developer_params)
+
+    if @developer.save
+      sign_in(@developer)
+      redirect_to(:board)
+    else
+      render(:new)
+    end
```
Comment (moklidia) [inline app/controllers/web/developers_controller.rb:14]:
```
return render(:new) unless @developer.save

sign_in(@developer)
redirect_to(:board)
```

**Pull request merged**